tools/generator/dfg/stm32/stm_device_tree.py

```diff
@@ -324,6 +324,26 @@ def rv(param, default=[]):
                 # print(instance, stream, channel)
                 # print(signal)
 
+        # Manually handle condition expressions from XML for
+        # (STM32F030CCTx|STM32F030RCTx) and (STM32F070CBTx|STM32F070RBTx)
```

Comment:
Horrible… I tried reverse-engineering that once but it's writting in Java and spread over many, many files. I got nowhere…